src/lib/ai/intervention/__tests__/RealTimeInterventionEngine.test.ts

```
@@ -0,0 +1,913 @@
+ * Comprehensive tests for the advanced real-time intervention system
+ */
+
+import { describe, it, expect, beforeEach, afterEach, vi } from 'vitest'
+import {
+  RealTimeInterventionEngine,
+  createRealTimeInterventionEngine,
+  performQuickRiskAssessment,
+  type RealTimeSession,
+  type RiskLevel,
+} from '../RealTimeInterventionEngine'
+import { EmotionDetectionEngine } from '../../emotions/EmotionDetectionEngine'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)